WordPress/Classes/ViewRelated/Reader/DetailWebView/ReaderWebView.swift

```diff
@@ -0,0 +1,96 @@
+import UIKit
+
+/// A WKWebView that renders post content with styles applied
+///
+class ReaderWebView: WKWebView {
+
+    /// Make the webview transparent
+    ///
+    override func awakeFromNib() {
+        isOpaque = false
+        backgroundColor = .clear
+    }
+
+    /// Loads a HTML content into the webview and apply styles
+    ///
+    @discardableResult
+    override func loadHTMLString(_ string: String, baseURL: URL?) -> WKNavigation? {
+        let content = """
+        <!DOCTYPE html><html><head><meta charset='UTF-8' />
+        <title>Reader Post</title>
+        <meta name='viewport' content='initial-scale=1, maximum-scale=1.0, user-scalable=no'>
+        <link rel="stylesheet" type="text/css"
+              href="https://wordpress.com/calypso/fallback/async-load-blocks-reader-full-post.feb2901cfbd98579fa81.min.css">
```

Comment:
In the final version, we won't have this link hardcoded here, even because that might change.

We just need to discover a better way to refer to this.